Formula/qt.rb

```diff
@@ -24,9 +23,9 @@ class Qt < Formula
   depends_on "cmake" => [:build, :test]
   depends_on "ninja" => :build
   depends_on "pkg-config" => :build
-  depends_on xcode: [:build, :test]
 
   depends_on "assimp"
+  depends_on "brotli"
```

Comment:
Do you need to change this?

https://github.com/Homebrew/homebrew-core/blob/539e9f5445d2bab4ce636a7f798bd94e5a8d9afe/Formula/qt.rb#L86